bin/adhoc-scripts/ParseSpecCmsswdist.py

```diff
@@ -254,12 +277,16 @@ def writeRequirements(specdir, depsspec, requirementsFilename):
 
 def main():
     """
-    :param spec_dir: Path to base direcotry that contains the spec files
-    :type spec_dir: str, required
-    :param spec_file: filename of the spec file
-    :type spec_file: str, required
+    All parameters taken through argument parsing:
+    :param --spec-dir: Path to base direcotry that contains the spec files
+    :type spec-dir: str, required
+    :param spec-file: filename of the spec file
+    :type spec-file: str, required
+    :param --level: Recursion level.
+    :param --all: A Bool flag to signal for building all dependencies not python related.
+    :param --recursive: A Bool flag to signal recursive iteration through spec files.
     """
-    logging.basicConfig(level=logging.INFO)
+    logging.basicConfig(level=logging.DEBUG, format='%(message)s')
```

Comment:
Do you want to have this change permanently in the script?

Reply:
fixed